config/v1/types_infrastructure.go

```diff
@@ -492,16 +518,42 @@ type OpenStackPlatformStatus struct {
 	// by components inside the cluster, like kubelets using the infrastructure rather
 	// than Kubernetes networking. It is the IP that the Infrastructure.status.apiServerInternalURI
 	// points to. It is the IP for a self-hosted load balancer in front of the API servers.
+	//
+	// Deprecated: Use APIServerInternalIPs instead. If APIServerInternalIPs and
```

Comment:
This field must continue to be valid and continue to be updated.  The choice about whether the new field takes precedence is not made by the setting entity, it is made by the consuming clients.

Reply:
I agree that the choice whether the new field takes precedence is made by the consuming clients. We added it to be more clear about the deprecation. Anyhow, updated it...